tests/tools/onert_train/src/onert_train.cc

```diff
@@ -145,6 +145,13 @@ int main(const int argc, char **argv)
       args.getLossReductionType().value_or(tri.loss_info.reduction_type);
     tri.opt = args.getOptimizerType().value_or(tri.opt);
 
+    size_t pos = 0;
+    tri.trainble_ops_size = args.getTrainableOpsIdx().size();
+    for (auto const &idx : args.getTrainableOpsIdx())
+    {
+      tri.trainble_ops_idx[pos++] = idx;
+    }
+
     std::cout << "== training parameter ==" << std::endl;
     std::cout << tri;
     std::cout << "========================" << std::endl;
```

Comment:
(suggestion)

Printing `trainable_ops` would be helpful for the user.   

https://github.com/Samsung/ONE/blob/313fb05beae628308da4b843118330bc71a4a999/tests/tools/onert_train/src/nnfw_util.cc#L106-L112